extensions-contrib/parquet-extensions/src/main/java/io/druid/data/input/parquet/GenericRecordAsMap.java

```diff
@@ -89,6 +91,9 @@ public boolean containsValue(Object value)
   public Object get(Object key)
   {
     Object field = record.get(key.toString());
+    if (field instanceof ArrayList) {
+      return Lists.transform((ArrayList) field, item -> String.valueOf(((GenericRecord) item).get(0)));
```

Comment:
Could you include a unit test for this, verifying the behavior works as expected?